### PR TITLE
installer: require Puppet 4.9 (RPM)

### PR DESCRIPTION
**Pull request by mmoll**
this should be built into:

* [x] Nightly

This is required since https://github.com/theforeman/puppet-tftp/commit/3a97422f61b37fd07e0583b9ad7eba2e4b66de0c and https://github.com/theforeman/puppet-git/commit/bb41057b854ca3338de276a3527d2b2c69c200e3

### CODE REVIEW

File: foreman-installer/foreman-installer.spec [hunk 1]
```diff
@@ -16,15 +16,15 @@ Source0:    http://downloads.theforeman.org/%{name}/%{name}-%{version}%{?dashalp
 BuildArch:  noarch
 
 Requires:   curl
-Requires:   %{?scl_prefix}puppet-agent >= 1.6.1
+Requires:   %{?scl_prefix}puppet-agent >= 1.9.0
```
Comment (ekohl) [inline foreman-installer/foreman-installer.spec]: Would puppet-agent be in the SCL? I know we build this without SCL so it's probably fine for now.

Comment (mmoll) [reply foreman-installer/foreman-installer.spec]: no idea.

Comment (ehelms) [reply foreman-installer/foreman-installer.spec]: It's not in the SCL and with the way its built I wouldn't ever forsee it being in so we could drop that.

**Comment by mmoll** [comment]
Could this get in for now? And do the de-SCLing later?

**Pull request merged**